backend/app/services/llm/providers/gai.py

```
@@ -155,6 +158,170 @@ def _execute_stt(
+            "speech_config": SpeechConfig(
+                voice_config=VoiceConfig(
+                    prebuilt_voice_config=PrebuiltVoiceConfig(voice_name=voice)
+                ),
+                language_code=language,
+            ),
+        }
+
+        if director_notes:
+            config_kwargs["system_instruction"] = director_notes
+
+        config = types.GenerateContentConfig(**config_kwargs)
```

Ruff (0.15.0)

[error] Undefined name `types`

(F821)